ci-operator/step-registry/openshift/release/fips/openshift-release-fips-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Get major.minor from payload tag
+major_minor=$(echo "${PAYLOAD_TAG}" | awk -F'.' '{print $1"."$2}')
```

Comment:
A word of caution about this PAYLOAD_TAG variable.  The release-controller has special logic to populate it only for "analysis" jobs.  So, you'll most likely need to rely on one of the variables passed into the steps:
https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables

Reply:
Yes, it'll contain the full PullSpec